phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -528,8 +528,9 @@ public SQLException newException(SQLExceptionInfo info) {
             "Cannot use a connection with SCN set to upsert data for a table with indexes."),
 
     CANNOT_PERFORM_DDL_WITH_PENDING_MUTATIONS(904, "43M15",
-            "Cannot perform DDL with pending mutations. Commit or rollback mutations before performing DDL");
+            "Cannot perform DDL with pending mutations. Commit or rollback mutations before performing DDL"),
 
+    NOT_SUPPORTED_CASCADE_FEATURE_PK(905, "43M16", "CASCADE feature is not supported to add PK column");
```

Comment:
Why does this have to be special error? I assume add PK column to phoenix table/view is not supported anyways. Also, if it has to be a new error then replace "CASCADE" with "CASCADE INDEX".